wordpress-mobile/WordPress-Android
# Reader Detail Updates (Update footer actions using the VM)

Task: #12900

This PR 
- Updates footer actions using the VM
- Removes now unused footer action methods (`refreshIconCount`, `setPostLike`, `togglePostLike` , `toggleBookmark`) from the fragment

**To Test**

1. Click a reader post
2. Verify that **click action on footer icons** behave correctly as before
3. Toggle like for the opened post from the website and verify that like counts are refreshed correctly on **pull to refresh** 
4. Verify that a **direct operation** (operation to perform automatically when opened via deeplinking) to like a post behaves correctly as before
(You can test it by replacing `directOperation` by `DirectOperation.POST_LIKE` on [this line](https://github.com/wordpress-mobile/WordPress-Android/blob/2455d927a42315687155e4e754d97026365a43d2/WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt#L1406)
`args.putSerializable(ReaderConstants.ARG_DIRECT_OPERATION, directOperation)`)

**Merge instructions**

1. Make sure #13129 is merged
2. Change branch to "issue/12900-rip3-detail-updates-main"
3. Remove "Not Ready for Merge"
4. Merge this PR

// TODO (added to backlog)
Add missing tracking
READER_POST_SAVED_FROM_DETAILS
READER_POST_UNSAVED_FROM_DETAILS
READER_SAVED_LIST_VIEWED_FROM_POST_DETAILS_NOTICE

PR submission checklist:
